class3.1/src/test.cpp

```diff
@@ -0,0 +1,64 @@
+#include <Helpers.hpp>
+#include <UnitTests.hpp>
+#include <gtest/gtest.h>  // я тут додала  Google Test
+
+
+int main() {
+    UnitTests testSuite;
+
+    testSuite.addTest("Dummy_test1", [](){
```

Comment:
Minor: it is better to have empty lines between logical steps of tests(Build-Operate-Check, Arrange-Act-Assert)
```
double value = 0.0;

auto result = dummyFunc(value);

ASSERT_EQ(result, 0.0)
```
It simplifies reading especially on bigger unit tests